packages/conform-yup/index.ts

```diff
@@ -98,7 +98,7 @@ export function parse<Schema extends yup.AnyObjectSchema>(
 		}: {
 			name: string;
 			intent: string;
-			payload: Record<string, any>;
+			payload: Record<string, unknown>;
```

Comment:
I changed these all to `unknown` so that it was clear we weren't handling the payload directly in library code and if there needed coersion it will warn in the future about potential improper data handling